Sources/Turf/Geometries/Polygon.swift

```diff
@@ -14,6 +14,28 @@ public struct Polygon: Equatable {
     public init(outerRing: Ring, innerRings: [Ring] = []) {
         self.coordinates = ([outerRing] + innerRings).map { $0.coordinates }
     }
+
+    /**
+     Returns a circular polygon based on a given radius and a defined
+     number of vertices for precision.
+
+     - Parameter center: The center coordinate for the circle.
+     - Parameter radius: The radius of the circle, measured in meters.
+     - Parameter vertices: The number of vertices the circle will have. Defaults to 64.
+     - Returns: A polygon resembling a circle.
+     */
+    public init(center: CLLocationCoordinate2D, radius: CLLocationDistance, vertices: Int = 64) {
```

Comment:
By describing it as a regular polygon initializer, we set expectations so that it isn’t strictly necessary to have a default value for the number of vertices. Most likely the client should explicitly choose the number of vertices anyways.